sql/plsql/PlSqlLexer.g4

```diff
@@ -2274,6 +2274,11 @@ SHA2_512_Q:                   '"SHA2_512"';
 SHARDED:                      'SHARDED';
 V1_Q:                         '"V1"';
 
+ISOLATE:                      'ISOLATE';
+ROOT:                         'ROOT';
+UNITE:                        'UNITE';
+ALGORITHM:                    'ALGORITHM';
```

Comment:
It looks like not all tokens added to `non_reserved_keywords_pre12c` but only `ROOT`.

Reply:
my bad